### PR TITLE
changelog

### PR DESCRIPTION
**Pull request by Kukks**
This was a long one :P

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
 
+## 1.0.5.0:
+
+### New Feature and improvements
+
+* Allow invoice and payment requests to be archived (hide from list) (@Kukks)
+* Improve fee selection UI in wallet send screen (@Kukks)
+* Show warning when enabling Payjoin but supported payment methods are not using a hot wallet (@Kukks)
+* Rebuild POS and Crowdfund App Item/Perk editor UI and fix any issues (@Kukks)
+* Adjust Payjoin to the new specification (@NicolasDorier)
```
Comment (NicolasDorier) [inline Changelog.md:11]: to BIP78

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+* Adjust Payjoin to the new specification (@NicolasDorier) 
+* Allow opening the BTCPay wallet natively when clicking a Bitcoin payment link (BIP21)
+* Add Server Info GreenField API (@dennisreimann)
+* Add Payment Requests GreenField API (@Kukks @NicolasDorier)
+* Support specifying payment method through apps per item/perk (@Kukks)
+* Allow setting private route hints for LND invoices (@r0ckstardev)
+* Expand GreenField Store API to have more store properties (@Kukks @NicolasDorier)
+* Make GreenField local docs require authentication (@Kukks)
+* Enhanced privacy when spending from BTCPay wallet (@NicolasDorier)
+* Randomize RBF support in BTCPay wallet by default for privacy (@NicolasDorier)
```
Comment (NicolasDorier) [inline Changelog.md:20]: You can add: `Privacy enhancement: Randomize wallet transaction fingerprint.`

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,53 @@
...
+* Add support for Cobo Vault wallet file import (@Kukks)
+* Split POS app cart and static and support both simulataenously (@dennisreimann)
```
Comment (NicolasDorier) [inline Changelog.md:22]: simultaenously

File: Changelog.md [hunk 4]
```diff
@@ -1,5 +1,53 @@
...
+* Add ThunderHub integration (@NicolasDorier)
+* Add option to attempt to create PSBT with full transaction for inputs to sign for Trezor (@Kukks)
+* Add Notifications feature (@r0ckstardev)
+### Bug fixes
+
+* Do not auto-complete generate wallet form (@Kukks)
+* Make sure copied notification is positioned correctly on checkout (@chewsta)
+* Fix broken documentation links (@Kukks @dennisreimann @jad0c)
+* Fix POS app item display issues (@bolatovumar)
+* Fix Invocie checkout modal close button theme issues (@bolatovumar)
```
Comment (NicolasDorier) [inline Changelog.md:36]: Invoice

**Comment by NicolasDorier** [comment]
I think you should group to second level (1st level = Feature/BugFixes, 2nd level Greenfield/UI?)

**Comment by NicolasDorier** [comment]
pull payment missing.
refund missing.

**Comment by Kukks** [comment]
> 
> 
> pull payment missing.
> refund missing.

not merged yet :P

File: Changelog.md (hunk 3 above)
Comment (dennisreimann) [inline Changelog.md:22]:
```suggestion
* Split POS app cart and static and support both simultaneously (@dennisreimann)
```

File: Changelog.md (hunk 1 above)
Comment (Kukks) [inline Changelog.md:11]:
```suggestion
* Adjust Payjoin to the new specification outlined in BIP78 (@NicolasDorier) 
```

File: Changelog.md [hunk 5]
```diff
@@ -1,5 +1,53 @@
...
+* Rebuild POS and Crowdfund App Item/Perk editor UI and fix any issues (@Kukks)
+* Adjust Payjoin to the new specification outlined in BIP78 (@NicolasDorier) 
+* Allow opening the BTCPay wallet natively when clicking a Bitcoin payment link (BIP21)
+* Add Server Info GreenField API (@dennisreimann)
+* Add Payment Requests GreenField API (@Kukks @NicolasDorier)
+* Support specifying payment method through apps per item/perk (@Kukks)
+* Allow setting private route hints for LND invoices (@r0ckstardev)
+* Expand GreenField Store API to have more store properties (@Kukks @NicolasDorier)
+* Make GreenField local docs require authentication (@Kukks)
+* Enhanced privacy when spending from BTCPay wallet (@NicolasDorier)
```
Comment (Kukks) [inline Changelog.md:19]:
```suggestion
* Privacy enhancement: Randomize wallet transaction fingerprint. (@NicolasDorier)
```

File: Changelog.md (hunk 4 above)
Comment (pavlenex) [reply Changelog.md]: > Add Notifications feature (@r0ckstardev) 

I think we should display significant features like this and pull payments before all the others.

Comment (Kukks) [reply Changelog.md]: I listed them based on commit date, we should reorder as we get closer to release

File: Changelog.md [hunk 6]
```diff
@@ -1,5 +1,53 @@
...
+* Randomize RBF support in BTCPay wallet by default for privacy (@NicolasDorier)
+* Add support for Cobo Vault wallet file import (@Kukks)
+* Split POS app cart and static and support both simultaneously (@dennisreimann)
+* Add Cross-Implementation Lightning Node GreenField API (@Kukks @NicolasDorier)
+* Make GreenField responses and JSON properties consistent (@Kukks @NicolasDorier)
+* Add Tor Onion-Location HTTP header (@dennisreimann) 
+* Rename form items in Wallet Send UI (@pavlenex)
+* Add ThunderHub integration (@NicolasDorier)
+* Add option to attempt to create PSBT with full transaction for inputs to sign for Trezor (@Kukks)
+* Add Notifications feature (@r0ckstardev)
```
Comment (pavlenex) [inline Changelog.md:29]: This needs to be displayed on top, not the bottom.

Comment (Kukks) [reply Changelog.md:29]: We'll re-arrange when we are close to release, better this way for now to keep track of what's new

Comment (pavlenex) [reply Changelog.md:29]: Good point.

File: Changelog.md [hunk 7]
```diff
@@ -1,5 +1,54 @@
...
+* Make sure copied notification is positioned correctly on checkout (@chewsta)
+* Fix broken documentation links (@Kukks @dennisreimann @jad0c @Eskyee @pavlenex)
+* Fix POS app item display issues (@bolatovumar)
+* Fix Invoice checkout modal close button theme issues (@bolatovumar)
+* Fix display of replaced transactions in invoice list (@NicolasDorier)
+* Support BitPay invoice creation property "paymentCurrencies" (@Kukks)
+* Update lightning support warning text (@janoside)
+* Fix issue with payment requests not expiring (@Kukks)
+* Fix broken Bootstrap theme generator link (@Kukks)
+* Use proper Bitcoin symbol (₿) in UI instead of "BTC" (@Kukks)
```
Comment (RiccardoMasutti) [inline Changelog.md:43]: Is it really necessary? It may not be visible to all devices

Comment (pavlenex) [reply Changelog.md:43]: Which devices it may not it be visible to, so we can test?

Comment (RiccardoMasutti) [reply Changelog.md:43]: The ₿ symbol appeared in [Unicode 10.0.0](https://www.unicode.org/versions/Unicode10.0.0/)

Comment (Kukks) [reply Changelog.md:43]: We should go ahead with release. If anyone runs into the issue we can easily revert. We can look into if a fallback is possible too

File: Changelog.md [hunk 8]
```diff
@@ -1,5 +1,58 @@
...
+* Add support for Wasabi wallet file import (@Kukks)
+* Split POS app cart and static and support both simultaneously (@dennisreimann)
+* Add Cross-Implementation Lightning Node GreenField API (@Kukks @NicolasDorier)
+* Make GreenField responses and JSON properties consistent (@Kukks @NicolasDorier)
+* Add Tor Onion-Location HTTP header (@dennisreimann) 
+* Rename form items in Wallet Send UI (@pavlenex)
+* Add ThunderHub integration (@NicolasDorier)
+* Add option to attempt to create PSBT with full transaction for inputs to sign for Trezor (@Kukks)
+* Add Notifications feature (@r0ckstardev)
+* Add Pull Payments feature (@NicolasDorier)
```
Comment (pavlenex) [inline Changelog.md:31]:
```suggestion
```

File: Changelog.md [hunk 9]
```diff
@@ -1,5 +1,58 @@
...
+* Add Refund feature (@NicolasDorier)
```
Comment (pavlenex) [inline Changelog.md:32]:
```suggestion
```

File: Changelog.md [hunk 10]
```diff
@@ -1,5 +1,58 @@
...
+* Add support for Cobo Vault wallet file import (@Kukks)
+* Add support for Wasabi wallet file import (@Kukks)
+* Split POS app cart and static and support both simultaneously (@dennisreimann)
+* Add Cross-Implementation Lightning Node GreenField API (@Kukks @NicolasDorier)
+* Make GreenField responses and JSON properties consistent (@Kukks @NicolasDorier)
+* Add Tor Onion-Location HTTP header (@dennisreimann) 
+* Rename form items in Wallet Send UI (@pavlenex)
+* Add ThunderHub integration (@NicolasDorier)
+* Add option to attempt to create PSBT with full transaction for inputs to sign for Trezor (@Kukks)
+* Add Notifications feature (@r0ckstardev)
```
Comment (pavlenex) [inline Changelog.md:30]:
```suggestion
```

File: Changelog.md [hunk 11]
```diff
@@ -1,5 +1,58 @@
 # Changelog
 
+## 1.0.5.0:
+
+### New Feature and improvements
+
```
Comment (pavlenex) [inline Changelog.md:6]:
```suggestion
* Add Notifications feature (@r0ckstardev)
* Add Pull Payments feature (@NicolasDorier)
* Add Refund feature (@NicolasDorier
```

**Pull request merged**